python/cudf/cudf/dataframe/numerical.py

```diff
@@ -286,6 +286,9 @@ def applymap(self, udf, out_dtype=None):
         )
         return self.replace(data=out, dtype=out_dtype)
 
+    def applymap_ptx(self, udf_ptx, np_dtype):
```

Comment:
Would be more Pythonic if we had a single public `applymap` function rather than the current `applymap` and `applymap_ptx` functions. We could have `applymap` dispatch to non-public `_applymap_jit` and `_applymap_ptx` functions based on the type of the `udf` argument.

Reply:
@shwina I don't know how to achieve the function overloading/dispatching you are mentioning in python though. I think @kkraus14 's idea is to fix the python interface in a separate PR.

Reply:
Sounds good